apps/api/src/app/step-schemas/dtos/step-schema.dto.ts

```diff
@@ -1,6 +1,83 @@
 import { JSONSchema } from 'json-schema-to-ts';
+import { StepType } from '@novu/shared';
+
+import { ActionStepEnum, ChannelStepEnum } from '@novu/framework/internal';
+
+const chatProperties = {
+  body: { type: 'chatBody' },
```

Comment:
I like the simple direction you've taken with prefixing the step type to the property name. It's opinionated and consistent and makes it easier for us to iterate on later.

Reply:
thanks i implemented the dx guide :)